cypress/e2e/models/administration/custom-migration-targets/custom-migration-target.ts

```diff
@@ -184,4 +191,9 @@ export class CustomMigrationTarget {
                 });
             });
     }
+
+    validateSourceTechnology(sources: string): void {
```

Comment:
@kpunwatk there could be more than one source, please use array

Reply:
Done, Thank you @mguetta1